app/api/llm/route.ts

```diff
@@ -146,15 +151,15 @@ export async function POST(request: NextRequest) {
       );
     }
 
-    const llm = new OpenAI({
-      model: config.model,
+    const llm = new Ollama({
```

Comment:
@thucpn how about if `config.model` is not one of `ALL_AVAILABLE_OPENAI_MODELS`, we fall back to `Ollama`?